gammapy/modeling/bayesian.py

```diff
@@ -0,0 +1,467 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from IPython.display import display
+import numpy as np
+from astropy.table import Table, vstack, Column
+from scipy.stats import gaussian_kde
+from .inference_data import inference_data_from_sampler
+
+
+class BayesianModelSelection:
+    """
+    Run Bayesian inference for a set of alternative models.
+
+    Parameters
+    ----------
+    datasets : `Datasets`
+        The datasets on which to perform the model fitting.
+    alternative_models : dict
```

Comment:
the parameters are not the one of the init function. A class has no return function